joeynmt/prediction.py

```diff
@@ -183,6 +183,7 @@ def test(cfg_file,
         try:
             step = ckpt.split(model_dir+"/")[1].split(".ckpt")[0]
         except IndexError:
+            # TODO: incorporate "last" as well 
```

Comment:
`get_latest_checkpoint` returns the last checkpoint

Reply:
Thanks for your comment :) Originally, I intended to have another save_ckpt() to control the ckpt saving by epoch frequency. Now, I add an argument "epoch_no" to _save_checkpoint() to differentiate ckpts saved by epoch_freq and validation_frequency

Reply:
Good solution :)